### ISSUE TITLE
[renderers branch] createCanvas() not initializing webgl renderer

### ISSUE THREAD

**Issue by indefinit** [processing/p5.js#580]
It looks like as of SHA: 8676369ad2076561ee166d0af7ba4e75d6070038 the webgl renderer is not being initialized through createCanvas.
eg.

```
// should create a webgl enabled canvas with id=defaultCanvas
createCanvas(displayWidth, displayHeight, 'webgl', true);
```

here's the line in rendering.js where this should be happening:
https://github.com/processing/p5.js/blob/renderers/src/rendering/rendering.js#L38

I remember it used to work back in December when @lmccart  merged SHA: 00b9a39bb2e64e19c2f157074a495dc3ba3d425a 

Should I try changing rendering.js back to its older commit state?

**Comment by lmccart** [comment]
oops did I break that? sorry... :\

if you want to take a look at fixing it back that'd be great!

**Comment by indefinit** [comment]
ok no prob.  Got some other goodies to commit in this branch anyways.  Just
didn't know if it was removed on purpose :)

On Mon, Mar 23, 2015 at 5:06 PM, Lauren McCarthy <notifications@github.com>
wrote:

> oops did I break that? sorry... :\
>
> if you want to take a look at fixing it back that'd be great!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/processing/p5.js/issues/580#issuecomment-85197436>.
>



-- 

Kevin Siwoff . data + sound + vision
// http://kevinsiwoff.com // http://studioindefinit.com

**Comment by lmccart** [comment]
oh cool! nope, definitely not on purpose... I was trying to bring this branch up to speed with master a little while ago and must have botched something there.

**Comment by indefinit** [comment]
Closing issue since this is fixed with the merge from p5jscon via #664.
